bundles/af-core/src/test/resources/schema/0.15.0/form.json

```diff
@@ -0,0 +1,14 @@
+{
+  "items": [
+  ],
+  "fieldType" : "form",
+  "metadata": {
+    "action": "/a/b",
+    "dataUrl": "/c/d",
+    "locale": "en-us",
+    "grammar": "json-formula-1.0.0",
+    "version": "1.0.0"
+  },
+  "adaptiveform":"0.15.0",
```

Comment:
0.15.2 is the latest version, please remove 0.15.0 from the resources

Reply:
Removed